### ISSUE TITLE
Depth loss confusion

### ISSUE THREAD

**Issue by zzwon1212** [pkqbajng/CGFormer#21]
Hello, thanks to your awesome work.

In your paper, you mentioned "We also employ explicit depth loss Ld [24] to supervise the estimated depth probability of the depth net."

But, [24] is [Bevformer](https://github.com/fundamentalvision/BEVFormer) which didn't use explicit depth network in my knowledge.

In your code, I found you specified `loss_depth_type='kld'` in [config file](https://github.com/pkqbajng/CGFormer/blob/main/configs/CGFormer-Efficient-Swin-SemanticKITTI-Pretrain.py).

Also, the below [code](https://github.com/pkqbajng/CGFormer/blob/main/mmdet3d_plugin/models/img2bev/GeometryDepth_Net.py) which gets depth loss, used `bce` loss or `kld` loss.

    @force_fp32()
    def get_depth_loss(self, depth_labels, depth_preds):
        if self.loss_depth_type == 'bce':
            depth_loss = self.get_bce_depth_loss(depth_labels, depth_preds)
        
        elif self.loss_depth_type == 'kld':
            depth_loss = self.get_klv_depth_loss(depth_labels, depth_preds)
        
        else:
            pdb.set_trace()
        
        return self.loss_depth_weight * depth_loss

So I have two questions
  1. What did you really use for depth loss?
  2. Which paper did you reference for depth loss?

**Comment by pkqbajng** [comment]
Hi, the citation should be [bevdepth](https://github.com/HuangJunJie2017/BEVDet/blob/dev3.0/mmdet3d/models/necks/view_transformer.py). In the released codebase and checkpoints, we employ depth loss. Recently, I found that using depth loss for only pretrain will obtain better results. I hope this can help you.
